### ISSUE TITLE
Save a mouse click when adding customer/project manually

### ISSUE THREAD

**Issue by paalolav** [Puzzlepart/did#1058]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Save a mouse click when adding customer/project manually.

### Description
When adding a customer/project manually, we have to select the search box to enter customer and/or project.
Taking us there, by the magic of computer wizardry, would save us a mouse click.
A click a day, keeps the doctor away, as she said, last night.

![Screenshot 2022-10-21 at 12 42 36](https://user-images.githubusercontent.com/55249575/197177863-dc9b4033-6de2-4df0-a01c-444b0a955b38.png)

Additionally, I'd love to be able to browse the result list from the "Search projects..."-box with my Up/Down keys and press Enter to select it. I know you can DO IT. <3
